float32[3] centerPoint
---
bool success
